docs/how-to/timeseries/sessionize.qmd

```diff
@@ -74,6 +74,6 @@ sessionized = (
 )
 ```
 
-Calling `ibis.show_sql(sessionized)` displays the SQL query and can be used to confirm that this Ibis table expression does not rely on any join operations.
+Calling `print(ibis.to_sql(sessionized))` displays the SQL query and can be used to confirm that this Ibis table expression does not rely on any join operations.
```

Comment:
Do we want to call out how to get syntax highlighting to show up in notebooks?

Reply:
Yep! It's just "remove the `print`" 😅

Reply:
Did this in `docs/index.qmd`.